index.js

```diff
@@ -18,11 +17,27 @@ const keyringTypes = [
   HdKeyring
 ]
 
-const ARGON2_TYPE = argon2.types.Argon2id
 const KEY_LENGTH = 32 // default key length in bytes
 const TextEncoder = window.TextEncoder || require('util').TextEncoder
 const TextDecoder = window.TextDecoder || require('util').TextDecoder
 
+
+const MODULE_CACHE = new WeakRef(arguments[5])
```

Comment:
this causes tests to fail because `WeakRef` doesn't exist in Node: https://travis-ci.com/github/brave/KeyringController/builds/213585178#L528. `window.WeakRef` might work. otherwise it probably needs to be polyfilled for the sake of tests.

Reply:
oof, good catch! I'll fix it right up